executor/scripts/local_executor_install.ps1

```
@@ -106,14 +108,83 @@ function Test-Npm {
+    try {
+        $codexOutput = & codex --version 2>$null
+        if ($codexOutput) {
+            $codexInstalled = $true
+            if ($codexOutput -match '\d+\.\d+\.\d+') {
+                $currentVersion = $Matches[0]
+            }
+        }
+    }
+    catch {
+        # Codex CLI not found
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)